### PR TITLE
Added platinum-bluetooth

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This simply adds platinum-bluetooth elements to platinum-elements

R= @jeffposnick @addyosmani

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
 # platinum-elements
-Elements that provide features to turn your web page into a true webapp - with things like push notifications, offline usage, and more.
+
```
Comment (wibblymat) [inline README.md]: Accidental extra newline?

Comment (beaufortfrancois) [inline README.md]: It was intentional actually :hamster: 
It looks better in a text editor in my opinion.

**Comment by jeffposnick** [comment]
The change LGTM, @beaufortfrancois, but please don't merge until https://github.com/PolymerElements/platinum-bluetooth is flipped public and you tag a `1.0.0` release. Otherwise, folks will start seeing the following error when they install the element set:

    bower platinum-bluetooth#^1.0.0          ENORESTARGET No tag found that was able to satisfy ^1.0.0

    Additional error details:
    No versions found in git://github.com/PolymerElements/platinum-bluetooth.git

**Comment by beaufortfrancois** [comment]
Thanks Jeffrey!
Can you quickly review/merge 2 simple PRs at https://github.com/PolymerElements/platinum-bluetooth/pulls so that I can tag and flip it?

**Comment by beaufortfrancois** [comment]
@jeffposnick https://github.com/PolymerElements/platinum-bluetooth is now public and tagged as `1.0.0` at https://github.com/PolymerElements/platinum-bluetooth/releases/tag/v1.0.0

Please merge at your earliest convenience.

**Comment by jeffposnick** [comment]
:+1:

**Pull request merged**